core/src/avm2/globals/flash/geom/Matrix3D.as

```diff
@@ -190,6 +190,14 @@ package flash.geom {
 			return new Vector3D(x, y, z, w);
 		}
 
+		public function transformVectors(vin:Vector.<Number>, vout:Vector.<Number>):void {
+			var result3D:Vector3D;
```

Comment:
Can you please add this code at the beginning:
```
if (vin == null) {
    throw new TypeError("Error #2007: Parameter vin must be non-null.", 2007);
}
if (vout == null) {
    throw new TypeError("Error #2007: Parameter vout must be non-null.", 2007);
}
```
Other than that, I think this PR is fine. Thanks for your contribution!